src/internet_identity/src/http/metrics.rs

```diff
@@ -152,6 +152,11 @@ fn event_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> Result<(), std::io::Error>
         "Cumulative authenticated session duration in seconds for the given front-end origin.",
     )?;
     pd_aggregation_metrics(PD_SESS_SEC, sess_sec_builder)?;
+    let count_builder = w.gauge_vec(
```

Comment:
Why wasn't this before exposed in /metrics ?

Reply:
Because there is non-negligible cost to the time-series kept in victoria metrics. So we only expose a specific subset to be available for historical data.

And this PR changes the subset to be more useful.

Reply:
Ah, ok, understood.